R/data.R

```diff
@@ -1,3 +1,38 @@
+#' variables.csv
+#' 
+#' This dataset lists all the variables that are present in cchsflow. 
+#' 
+#' See \href{https://big-life-lab.github.io/cchsflow/articles/variablesSheet.html}{here} for more
+#' details about how the worksheet is structured
+#' 
+#' @name variables
+#' @aliases variables
+#' @docType data
+#' @return \item{variables}{a data frame}
+#' 
+#' @keywords datasets
+#' @examples 
+#' data(variables)
+#' str(variables)
+NULL
+
+#' variableDetails.csv
+#' 
+#' This dataset provides details on how variables are specified in cchsflow.
+#' 
+#' See \href{https://big-life-lab.github.io/cchsflow/articles/variablesDetails.html}{here} for more
```

Comment:
I'm a bit concerned that we'll have people at secure data centres like ICES and Statscan that can't click on these links. They'll have a second monitor open and have a laborious process of copy links, emailing to their unsecure network, etc. 

I'm not sure there is an easy solution other than increasing the amount of information in this file. 

what I like is that the link is pretty easy to navigate if you're on the website.

@deirdrehennessy Thoughts?